composer.json

```diff
@@ -7,6 +7,14 @@
         "jonom/silverstripe-betternavigator": {
             "url": "https://github.com/unclecheese/silverstripe-betternavigator.git",
             "type": "git"
+        },
+        "silverstripe/bambusa-selfservice": {
```

Comment:
Once this approach and PR is accepted, we should move those repos to `silverstripe`. We've already made the decision to keep bambusa as a public repo in github.com/silverstripe rather than github.com/silverstripeltd, so any functionality separated out from that existing repo should follow the same rationale.